src/utilities/file.service.ts

```diff
@@ -1,7 +1,10 @@
 import { readdir, readFile } from 'fs/promises';
 import { HttpException, HttpStatus, Injectable, Logger, StreamableFile } from '@nestjs/common';
-import { PathLike } from 'fs';
+import { existsSync, mkdirSync, PathLike, readdirSync, rmSync, readFileSync } from 'fs';
 import * as xml2js from 'xml2js';
+import { getDocument } from 'pdfjs-dist/legacy/build/pdf';
+
+const pdf = require('pdf-poppler');
```

Comment:
Import * as pdf from 'pdf-poppler'

Reply:
This won't work, consult [this](https://webcache.googleusercontent.com/search?q=cache:90bdkKa5wmwJ:https://stackoverflow.com/questions/34563578/webpack-typescript-require-works-but-import-doesnt/34569845+&cd=1&hl=en&ct=clnk&gl=us) stack overflow post, or the original GitHub issue that raised this issue **here** [here](https://github.com/Microsoft/TypeScript/issues/247)